jekyll/_cci2/ios-migrating-from-1-2.md

```diff
@@ -0,0 +1,386 @@
+      # and save the same XML files under test-results folder
+      # in the Artifacts tab.
+      - store_test_results:
+          path: test_output/report.xml
+      - store_artifacts:
+          path: /tmp/test-results
+          destination: scan-test-results
+      - store_artifacts:
+          path: ~/Library/Logs/scan
+          destination: scan-logs
+
+  deploy:
+    macos:
+      xcode:
+        version: 8.3.3
+
+    steps:
+      - checkout
+
+      # Set up code signing via Fastlane Match.
+      - run:
+          name: Set up code signing
+          command: fastlane match development --readonly
+
+      # Build the release version of the app.
+      - run:
+          name: Build IPA
+          command: fastlane gym
+
+      # Store the IPA file in the build artifacts
+      - store_artifacts:
+          path: ./MyApp.ipa
+          destination: ipa
+
+      # Deploy!
+      - run:
+          name: Deploy to App Store
+          command: fastlane spaceship
+
+workflows:
+  version: 2
+  build-and-deploy:
+    jobs:
+      - build-and-test
+      - deploy:
+          requires:
+            - build-and-test
+          filters:
+            branches:
+              only: master
+```
+
+## Before you begin
+
+To ensure a consistent build experience, we recommend adding a Gemfile
+and setting up code signing with the help of Fastlane Match before you
+push a 2.0 config to your CircleCI iOS project.
+
+### Gemfile
+
+We suggest that you add a Gemfile to your iOS project if you don't have
+one already. This will ensure that recent versions of Fastlane and
+CocoaPods are available in your build.
+
+A sample Gemfile could look like this:
+
+```
+# Gemfile
+
+source 'https://rubygems.org'
+gem 'fastlane'
+gem 'cocoapods'
+```
+
+### Credentials for code signing via Fastlane Match
+
+For 2.0 we don't support automated code signing, and instead we recommend
+using Fastlane Match to manage code signing certificates in your iOS projects.
+
+It is best to set up code signing once you have a successful `build` job
+in your iOS project on CircleCI.
+
+Check out the Fastlane Match [getting started guide](https://codesigning.guide/)
+for the exact steps for setting up a Match repo and storing your
+certificates in it.
+
+Once your certificates are uploaded, you'll need to grant CircleCI
+permissions to access your certificates repo on GitHub. You can do that
+by going to your CircleCI Project Settings -> Checkout SSH Keys -> Add
+User Key -> Authorize with GitHub.
+
+Please bear in mind that this will allow CircleCI access to _all_ of
+your private repos.
+
+Once you have configured the User Key in the project settings, CircleCI
+will be able to fetch the certificates from GitHub.
+
+## Copying your build commands to the 2.0 config file
+
+### Job name and Xcode version
+
+In the 2.0 config file the first few lines specify the name of the job
+we will be running for you, and the Xcode version to use:
```

Comment:
change to: In the 2.0 `.circleci/config.yml` file, the first few lines specify the name of the job and the Xcode version to use: